templates/sysv/default/init.sh

```diff
@@ -43,12 +43,10 @@ start() {
       program we intended to run. Luckily, the 'chroot' program on OSX, FreeBSD, and Linux
       all support switching users and it invokes execve immediately after chrooting. }}
 
-  {{#sysv_log_directory?}}
   # Ensure the log directory is setup correctly.
-  [ ! -d "{{{sysv_log_path}}}" ] && mkdir "{{{sysv_log_path}}}"
-  chown "$user":"$group" "{{{sysv_log_path}}}"
-  chmod 755 "{{{sysv_log_path}}}"
-  {{/sysv_log_directory?}}
+  [ ! -d "{{{ log_directory }}}" ] && mkdir "{{{ log_directory }}}"
+  chown "$user":"$group" "{{{ log_directory }}}"
```

Comment:
This will try to chown the directory (default, /var/log) to be owned by the user this service runs as. I think this is maybe not the right solution? I see the older code did this too, but I think the older code defaulted to something like /var/log/[program]/ which was slightly less wild, though I might be mistaken.

Maybe instead of chown'ing the directory, we could `touch` the target log files and chown them to be $user:$group?